PINRemoteImage.xcodeproj/project.pbxproj

```diff
@@ -1105,6 +1143,7 @@
 		68A0FC221E523434000B552D /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
```

Comment:
Are we sure we want this to be YES?

Reply:
No 😅
All of those are changes automatically done by Xcode
Do you want me to revert them and leave only those that introduced the new files?

Reply:
Offline discussion took place, we decided to leave all and unset `ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES`